internal/kafka/internal/clusters/types/types.go

```diff
@@ -88,6 +88,12 @@ type ClusterSpec struct {
 	StatusDetails string `json:"status_details"`
 	// additional information related to the cluster, can vary depending on the provider
 	AdditionalInfo api.JSON `json:"additional_info"`
+	// cloud provider region the cluster
+	Region string `json:"region"`
```

Comment:
If new attributes are added here, all consumers of that type might make use of them. Are we setting that properly for those elements?
Quickly reviewing it seems that might not be the case

Reply:
+1. I'll see what can be done here.